.pre-commit-config.yaml

```diff
@@ -23,39 +23,39 @@ repos:
       - id: debug-statements
 
   - repo: https://github.com/pycqa/pydocstyle.git
-    rev: 5.1.1
+    rev: 6.1.1
     hooks:
       - id: pydocstyle
 
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.4.0
+    rev: v4.1.0
     hooks:
       - id: check-toml
       - id: check-yaml
       - id: end-of-file-fixer
       - id: trailing-whitespace
 
   - repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
my suggestion for now. 
let's remove mypy out of pre-commit and can add it as a separate check like this https://github.com/thoth-station/messaging/blob/82588c945130172f0db76d81a55ea015dabea79e/.prow.yaml#L22
so our pr is not on hold just for mypy